package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "wthr",
+  "private": true,
+  "version": "0.0.0",
```

The version should be the same across the entire project. Currently, the branch is `v1.0`.
```suggestion
  "version": "1.0",
```

---

?